### ISSUE TITLE
Curl imports don't correctly handle escaped characters from the --data-binary field

### ISSUE THREAD

**Issue by gs-nkumar** [a85/POSTMan-Chrome-Extension#4772]
![postman](https://user-images.githubusercontent.com/17634822/42269996-6139d3d0-7f9d-11e8-9f03-3bbd7362923a.jpg)

**Comment by numaanashraf** [comment]
@gs-nkumar How did you create this request in Postman? Is it via Copy as cURL in Chrome DevTools (Network) and then `Import` in Postman by `pasting raw text`?

**Comment by gs-nkumar** [comment]
yes

**Comment by numaanashraf** [comment]
@gs-nkumar Could you provide the raw text you had pasted into the `import` dialog here?

**Comment by gs-nkumar** [comment]
```
curl 'https://abstract.com/v1/tenant/' -H 'appOrgId: ' -H 'Origin: http://localhost:4200' -H 'appUserId: ' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) ' -H 'content-type: application/json' -H 'Accept: application/json, text/plain, */*' -H 'Referer: http://localhost:4200/' -H 'appSessionId: ' -H 'authToken: authotoken' --data-binary $'{\n  "entityType": "COMPANY",\n  "relationshipTypeId": null,\n  "name": "my survey",\n  "languages": [\n    "1I002G8V5KV3GDIJU1FLKIKTXMZRK51WN12W"\n  ],\n  "partialSubmitDays": 10\n}' --compressed
```

**Comment by numaanashraf** [comment]
@gs-nkumar Thanks. This helps a lot. I'm able to replicate this issue. We'll update here when the fix is out.

**Comment by gs-nkumar** [comment]
just added my personal profile to track this.@nbpro

**Comment by VShingala** [comment]
@gs-nkumar This issue is now fixed with the latest postman app v9.1.2.
